### PR TITLE
[14.0][ADD] account_move_line_landed_cost_info

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
This module will add the landed cost info to journal items.

The ultimate goal is to establish the landed cost adjustments as one of the key
fields to reconcile the Goods Delivered Not Invoiced accrual account.

@ForgeFlow

### CODE REVIEW

File: account_move_line_landed_cost_info/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    _inherit = "account.move.line"
+
+    stock_valuation_adjustment_line_id = fields.Many2one(
+        comodel_name="stock.valuation.adjustment.lines",
+        string="Stock Valuation Adjustment Line",
+        store=True,
+        index=True,
+    )
+    cost_id = fields.Many2one(
```
Comment (JordiBForgeFlow) [inline account_move_line_landed_cost_info/models/account_move.py]: change to 'stock_landed_cost_id'

**Comment by AaronHForgeFlow** [comment]
The hook was successfully used in an actual database

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
It seems the merge command did not run. I just rebased the branch to see if CI is still green.

**Comment by AaronHForgeFlow** [comment]
Not sure how to fix the error in the runboat:

```
Collecting vatnumber (from -r test-requirements.txt (line 11))
  Downloading vatnumber-1.2.tar.gz (19 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in vatnumber setup command: use_2to3 is invalid.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Pull request merged**